docker/hosted-mode-tls/docker-compose.yaml

```
@@ -17,7 +17,7 @@ services:
       - "../../spec_files:/opt/mojaloop-testing-toolkit/spec_files"
       - "../../secrets:/opt/mojaloop-testing-toolkit/secrets"
```

```suggestion
      - "../../secrets:/opt/app/secrets"
```

Note: please check for any other places where the directory will need to be updated